src/components/MoviesSearchBar/MoviesSearchBar.js

```diff
@@ -1,7 +1,24 @@
-import React from "react";
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import { searchMoviesData } from "../../store/moviesActions";
 import "./MoviesSearchBar.css";
 
 const MoviesSearchBar = () => {
+  const [query, setQuery] = useState("");
+  const dispatch = useDispatch();
+
+  const handleKeypress = (event) => {
+    if (event.keyCode === 13) {
```

Comment:
You should not have magic numbers in your code. Move 13 to const NAME_OF_A_CONST